src/react-components/home/HomePage.js

```diff
@@ -50,12 +51,27 @@ export function HomePage() {
   }, []);
 
   const canCreateRooms = !configs.feature("disable_room_creation") || auth.isAdmin;
-
+  const email = auth.email;
   return (
     <PageContainer className={styles.homePage}>
       <Container>
         <div className={styles.hero}>
-          <SignInButton mobile />
+          {auth.isSignedIn ? (
```

Comment:
If we find ourselves copying this code some more we might want to turn it into a component that handles both logged in and out states.